src/doc/rustdoc/src/how-to-read-rustdoc.md

```diff
@@ -0,0 +1,101 @@
+# How to read rustdoc output
+
+Rustdoc's HTML output includes a friendly and useful navigation interface,
+to make it easier for users to navigate and understand your code.
+This chapter covers the major features of that interface,
+and is a great starting point for documentation authors and users alike.
+
+## Structure
+
+The `rustdoc` output is divided into three sections.
+Along the left side of each page is a quick navigation bar,
+which shows contextual information about the current entry.
+The rest of the is taken up by the search interface at the top
+and the documentation for the current item below that.
+
+## The Item Documentation
+
+The majority of the screen is taken up with the main event:
+the documentation text for the item currently being viewed.
+The main title is the type and name of the item,
+such as "Struct `std::time::Duration`",
+along with a button to collapse or expand the top-level documentation for that
+item (`[+]` or `[-]`),
+a link to the source code (`[src]`),
+if [configured](the-doc-attribute.html#html_no_source),
+and the version in which the item became stable,
+if it's a stable item in the standard library.
+
+Below this is the main documentation for the item,
+including a definition or function signature if appropriate,
+followed by a list of fields or variants for structs, unions, and enums.
```

Comment:
Instead of listing the different types, wouldn't it be better to simply say "for rust types"?

Reply:
Seems reasonable!